toolchain/openocd-mimiker/Makefile

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/make -f
+
+VER := 0.10.0+dev
+OPENOCD := openocd-$(VER)
+
+all: binary
+
+download:
+	git clone https://git.code.sf.net/p/openocd/code $(OPENOCD)
+
+configure:
+	rm -rf openocd-build
+	mkdir openocd-build
+	cd $(OPENOCD) && \
+	    ./bootstrap
+	cd openocd-build && \
+	    ../$(OPENOCD)/configure \
+	        --prefix=/usr \
+		--enable-jlink 
+	touch $@-stamp
```

Comment:
Even though you create stamps this (and others too) rule will trigger again if you run `make` twice.

Reply:
Added intermediate step to properly generate stamp files.